CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.3.0] 2022-08-16
+
+### Added
+- Added html import plugin tab, its settings, and data flow
+
+### Changed
+- html import configuration
```

Comment:
```suggestion
- html import plugin configuration
```
mehet ez is az added-be, mert ha jól emlékszem minden csak poc szinten volt benne eddig kikommentelve, ami nem számit